docs/_plugins/build_api_docs.rb

```diff
@@ -0,0 +1,205 @@
+#
```

Comment:
We should probably change:

```
dev/change-scala-version.sh:echo "$BASEDIR/docs/_plugins/copy_api_dirs.rb"
dev/change-scala-version.sh:  sed_i '/\-Pscala-'$TO_VERSION'/!s:build/sbt:build/sbt \-Pscala\-'$TO_VERSION':' "$BASEDIR/docs/_plugins/copy_api_dirs.rb"
dev/change-scala-version.sh:  sed_i 's:build/sbt \-Pscala\-'$FROM_VERSION':build/sbt:' "$BASEDIR/docs/_plugins/copy_api_dirs.rb"
dev/change-scala-version.sh:sed_i 's/scala\-'$FROM_VERSION'/scala\-'$TO_VERSION'/' "$BASEDIR/docs/_plugins/copy_api_dirs.rb"
```

together ..

Reply:
Oh, good catch. Will update.

Reply:
Fixed and updated the PR description.